src/coreclr/jit/flowgraph.cpp

```diff
@@ -1067,7 +1062,8 @@ GenTree* Compiler::fgOptimizeDelegateConstructor(GenTreeCall*            call,
     }
     else if (oper == GT_CALL && targetMethod->AsCall()->gtCallMethHnd == eeFindHelper(CORINFO_HELP_VIRTUAL_FUNC_PTR))
     {
-        GenTree* handleNode = targetMethod->AsCall()->gtCallArgs->GetNext()->GetNext()->GetNode();
```

Comment:
Glad to see such `->GetNext()->GetNext()->GetNode()` are removed 😄

Reply:
Indeed, although I will have to review these carefully a few times since 'this' is now a regular arg, so some of these might have shifted.